src/delay/readme.md

```diff
@@ -11,7 +11,7 @@ import { delay } from 'patronum/delay';
 ### Formulae
 
 ```ts
-target = delay({ source, timeout: number, target });
+event = delay({ source, timeout: number, target });
```

Comment:
```suggestion
target = delay({ source, timeout: number, target });
```

or change the name of `target` in Formulae

Reply:
I intentionally changed it to `event` as `delay` does not return `typeof target`, instead, it returns `timerFx.doneData`, totally unrelated to the passed target. This might cause misunderstanding, so I explicitly named the returned event differently

If you think this will not be an issue, will gladly change back to `target` everywhere

Reply:
Actually, returned `target` must be the same as `target` from the arguments.

Reply:
This sounds like a breaking change. But I'll look into it and update the PR